pkgs/development/python-modules/poetry-core/default.nix

```diff
@@ -27,6 +29,20 @@ buildPythonPackage rec {
     hash = "sha256-h3d0h+WCrrNlfPOlUx6Rj0aG6untD6MiunqvPj4yT+0=";
   };
 
+  # revert update of vendored dependencies to unbreak e.g. zeroconf of x86_64-darwin
+  patches = lib.optionals (stdenv.isDarwin && stdenv.isx86_64) [
```

Comment:
We don't want to apply those on all platforms? It would avoid bit rotting when updating.

Reply:
We need to take this upstream either way. We can't keep applying this patch forever. For now it prevents a hefty number of rebuilds on the other three platforms. Also we don't know if this breks anything else.